src/server/req_manager.c

```diff
@@ -167,6 +167,8 @@ extern char *msg_cannot_set_route_que;
 
 extern int check_req_aoe_available(struct pbsnode *, char *);
 int resize_prov_table(int);
+int remove_mom_ipaddresses_list(mominfo_t *pmom);
```

Comment:
No need for this.  You don't actually call the function in this file.

Reply:
That's true. I will remove it.